docs/context.rst

```diff
@@ -21,50 +21,67 @@ To override the ``ContextManager`` you will need to override the ``getContextMan
 method in the ``DefaultSentryClientFactory``. A simpler API will likely be provided in
 the future.
 
-Using Breadcrumbs
------------------
+Usage
+-----
 
 Breadcrumbs can be used to describe actions that occurred in your application leading
 up to an event being sent. For example, whether external API requests were made,
-or whether a user clicked on something in an Android application.
+or whether a user clicked on something in an Android application. By default the last
+100 breadcrumbs per context will be stored and sent with future events.
 
-Once a ``SentryClient`` instance has been initialized, either via a logging framework or manually,
-you can begin recording breadcrumbs. By default the last 100 breadcrumbs for a given
-context instance will be stored and sent with future events.
+The user can be set per context so that you know who was affected by each event.
+
+Once a ``SentryClient`` instance has been initialized you can begin setting state in
+the current context.
 
 .. sourcecode:: java
 
     import io.sentry.Sentry;
     import io.sentry.context.Context;
     import io.sentry.event.BreadcrumbBuilder;
-    import io.sentry.event.Breadcrumbs;
     import io.sentry.event.UserBuilder;
 
-    public void example() {
-        // Record a breadcrumb without having to look up the context instance manually
-        Sentry.record(
-            new BreadcrumbBuilder().setMessage("User did something specific again!").build()
-        );
+    public class MyClass {
+        public void staticAPIExample() {
```

Comment:
It might be helpful to include a short note about when you'd want to use static and instance-based approaches just so people are less likely to copy/paste and then wonder why they're getting breadcrumbs interleaved from multiple requests, etc.